src/store/index.js

```diff
@@ -6,5 +6,4 @@ import rootReducer from '../reducers';
 
 const store = createStore(rootReducer, composeWithDevTools(applyMiddleware(thunk)));
 
-
```

Comment:
What was the file change here for

Reply:
I deleted an extra line at the end of the file